### ISSUE TITLE
Predicting with my own model

### ISSUE THREAD

**Issue by zhao98** [MIC-DKFZ/TractSeg#132]
I have preprocessed the HCP data, and I have tried to train my own model. I used `ExpRunner --train=False --test=False --seg --config=My_custom_experiment` to test my trained model. I have several problems now. Would you help me with these please?

1. The results seem that segmentation is failed. I think the training process was reasonable. The loss can converge normally after 219 epochs. Do you think there might be any reason for the failure of the segmentation? 

2. Is it possible that I failed to use the model I had trained? How can I make sure that my model is used in testing?

3. Could I use `TractSeg -i peak_file -o dir --output_type tract_segmentation`  to test my own model? If possible, how to implement this?

**Comment by wasserth** [comment]
I would recommend to use `TractSeg -i peak_file -o dir --output_type tract_segmentation` to test your own model. For this command to use the weights of the model you trained yourself you have to put your own weights here: `~/.tractseg/pretrained_weights_tract_segmentation_v3.npz`

**Comment by zhao98** [comment]
I have used `TractSeg -i peak_file -o dir --output_type tract_segmentation --exp_name My_custom_experiment` to test my model successfully. Thanks!

**Comment by wasserth** [comment]
Using `--exp_name <name>` also works and is actually easier. I forgot about this option ;). Have not used it in a while.
